src/components/BCheckbox/BCheckbox.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <!-- Group checkboxes: requires group id and label -->
-  <div v-if="groupId && groupLabel" class="column">
-    <div role="group" :aria-labelledby="groupId" class="flex">
-      <label :id="`group-${groupId}`" :class="{ bold: true, disabled: disabled }">
+  <div v-if="groupId && groupLabel" class="b-checkbox b-flex-column">
```

Comment:
Same with `b-flex-column`.

Reply:
Was `b-flex-column` already a class because you added the `b-flex-` to it in the HTML but not in the CSS.